src/test/run-make/raw-dylib-alt-calling-convention/Makefile

```diff
@@ -1,12 +1,12 @@
 # Test the behavior of #[link(.., kind = "raw-dylib")] with alternative calling conventions.
 
-# only-i686-pc-windows-msvc
+# only-i686-pc-windows-gnu
```

Comment:
These tests are supposed to eventually test both architectures, right?

Reply:
Yeah, I just didn't want to make spend time on proper fix yet.